### PR TITLE
February meetup post

### PR DESCRIPTION
**Pull request by jdamerow**

### CODE REVIEW

File: content/news/2024-01-25-announcing-february-meetup.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+author: DHTech
+thumbnail: /images/posts/meetup.jpeg
+featureImage: /images/posts/meetup.jpeg
+featureImageCap: Photo by <a href="https://unsplash.com/@charlesdeluvio?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">charlesdeluvio</a> on <a href="https://unsplash.com/photos/wn7dOzUh3Rs?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+
+tags:
+    - meetup
+---
+
+# DHTech February Meetup on Agend-based Modeling
```
Comment (ColeDCrawford) [inline content/news/2024-01-25-announcing-february-meetup.md:14]: Agent-based?

File: content/news/2024-01-25-announcing-february-meetup.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Our next DHTech meetup will be on February 15, 2024 at 9am ET/3pm CET. We will talk about agend-based modeling (ABM)! Are you using agent-based models for a project? Or are you interested in learning more about ABM? Do you have questions you would like to discuss like how to best design and calibrate a model? Come join us on Zoom! You can registered [here](https://asu.zoom.us/meeting/register/tZElduuprjsvEtFkc0xgDwpm5CVREJaBQDNT).
```
Comment (ColeDCrawford) [inline content/news/2024-01-25-announcing-february-meetup.md:16]: agent-based

**Review by ColeDCrawford — CHANGES_REQUESTED** [review]

File: content/news/2024-01-25-announcing-february-meetup.md (hunk 1 above)
Comment (jdamerow) [reply content/news/2024-01-25-announcing-february-meetup.md:14]: ups 😝

File: content/news/2024-01-25-announcing-february-meetup.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+featureImage: /images/posts/meetup.jpeg
+featureImageCap: Photo by <a href="https://unsplash.com/@charlesdeluvio?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">charlesdeluvio</a> on <a href="https://unsplash.com/photos/wn7dOzUh3Rs?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+
+tags:
+    - meetup
+---
+
+# DHTech February Meetup on Agent-based Modeling
+
+Our next DHTech meetup will be on February 15, 2024 at 9am ET/3pm CET. We will talk about agent-based modeling (ABM)! Are you using agent-based models for a project? Or are you interested in learning more about ABM? Do you have questions you would like to discuss like how to best design and calibrate a model? Come join us on Zoom! You can registered [here](https://asu.zoom.us/meeting/register/tZElduuprjsvEtFkc0xgDwpm5CVREJaBQDNT).
```
Comment (rlskoeser) [inline content/news/2024-01-25-announcing-february-meetup.md:16]: noticed a small typo

```suggestion
Our next DHTech meetup will be on February 15, 2024 at 9am ET/3pm CET. We will talk about agent-based modeling (ABM)! Are you using agent-based models for a project? Or are you interested in learning more about ABM? Do you have questions you would like to discuss like how to best design and calibrate a model? Come join us on Zoom! You can registere [here](https://asu.zoom.us/meeting/register/tZElduuprjsvEtFkc0xgDwpm5CVREJaBQDNT).
```

File: content/news/2024-01-25-announcing-february-meetup.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: DHTech February Meetup
```
Comment (rlskoeser) [inline content/news/2024-01-25-announcing-february-meetup.md:3]: should we put agent-based modeling in the title?
```suggestion
title: DHTech February Meetup on Agent-Based Modeling
```

**Comment by jdamerow** [comment]
Clearly, I wrote this really quick without paying much attention... 😆 Thanks for the proof-reading!

**Pull request merged**